source/testing_govuk_pay/index.html.md.erb

```diff
@@ -3,105 +3,65 @@ title: Testing GOV.UK Pay
 weight: 90
 ---
 
-# Testing GOV.UK Pay
+# Testing your integration
 
-When your GOV.UK Pay account is made live, you will still have a ‘test’
-account for general testing and experimenting with different settings and
-features. You can read more in the [__Switching to
-live__](/switching_to_live/#switching-to-live) section.
+### Testing your service
 
-When you test, you should make sure that you use [mock card
-numbers](#submit-a-test-transaction-using-mock-card-numbers) to simulate both successful
-and unsuccessful transactions. You must use test cards with your test account.
-Real card numbers will not work.
+You should:
```

Comment:
I know this is a style guidance thing, but I find the starkness of the "you should"/"you should not" doesn't leave room to describe the context.

I'd prefer to say something like:

You'll typically want to build the following kinds of tests:

1. developer focused tests that check the correctness of your integration with our API specifications
2. monitoring checks such as smoke tests that continuously monitor connectivity and integration compatibility

If you expect moderate or high volumes of payments (more than 2000 payments per day), you may also wish to carry out some performance testing.

For developer focused tests, you should build your own stub of GOV.UK Pay's api using a tool such as wiremock or mountebank. 

For smoke testing you can use our test account.

For performance testing, please contact us to make arrangements. Please do not run performance tests against your test or live account.

Reply:
Hi @heathd - thanks for working through this with me. The version we worked on yesterday is now in https://github.com/alphagov/pay-tech-docs/pull/326/commits/041acc8d5a90525b0006afc2e2f2157fdd086b5e.

I had a think about this and unfortunately I don't think we can add a rate limit number for test accounts, because the lower rate limit isn't enacted yet. It would also mean updating https://docs.payments.service.gov.uk/api_reference/#rate-limits, which would be a tricky prospect if it was a mix of current and future rate limits.

However I think we still get across what users need to know (not using your test account for high-rate integration testing) without the specific test account rate limit. Have a look and let me know what you think!

Reply:
👍 I still think that having some of the "why" as well as the "what" would be valuable (why: avoid flakiness, don't overload our production system) but I think this is a big improvement over what we have now so let's ship it

Reply:
Thanks David! Let's definitely chat about the "why" for the next iteration - I think you're right that we could do with more of it here.